docs/extend/plugin_api.md

```diff
@@ -14,9 +14,13 @@ weight=1
 Docker plugins are out-of-process extensions which add capabilities to the
 Docker Engine.
 
+This document describes the Docker Engine plugin API generally available in
+Docker Engine 1.12 and earlier. To view information on plugins managed by Docker
+Engine, refer to [Docker Engine plugin system](plugins.md).
```

Comment:
Since legacy plugins are still available, perhaps we should reverse this sentence. "For documentation on the managed plugin system introduced in Docker engine 1.13, refer to..."

(Using 1.13, not sure if we should use 1.12, as it was only available in the release with "experimental" features enabled)